src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java

```diff
@@ -0,0 +1,247 @@
+package org.geoserver.web;
+
+import static org.junit.Assume.assumeTrue;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.List;
+import net.sf.json.JSONObject;
+import org.apache.commons.io.IOUtils;
```

Comment:
This failed the QA checks:

```
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTestSupport.java:[85,34] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java:[91,34] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java:[130,31] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java:[161,31] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java:[185,31] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
[WARNING] /home/travis/build/geoserver/geoserver/src/web/core/src/test/java/org/geoserver/web/GeoServerWicketOnlineTest.java:[215,34] toString(java.io.InputStream) in org.apache.commons.io.IOUtils has been deprecated
```

Reply:
Huh, odd, worked for me locally. That should probably be the GeoTools IOUtils anyways - I'll look into it.